rigex-front-end/src/components/auth/signUpForm.js

```diff
@@ -0,0 +1,203 @@
+import React, { useState, useEffect } from 'react';
+import { gql, useMutation } from '@apollo/client';
+import { useDispatch } from 'react-redux';
+
+import Button from '@material-ui/core/Button';
+import IconButton from '@material-ui/core/IconButton';
+import Input from '@material-ui/core/Input';
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from '@material-ui/core/InputLabel';
+import Mail from '@material-ui/icons/Email';
+import Lock from '@material-ui/icons/Lock';
+import VisibilityOutlined from '@material-ui/icons/VisibilityOutlined';
+import VisibilityOffOutlined from '@material-ui/icons/VisibilityOffOutlined';
+import Grid from '@material-ui/core/Grid';
+
+import './signUpForm.scss';
+import { signUpActions } from '../../store/signUp/action';
+import logo from '../../assets/large-logo.png';
+import history from '../../utils/history';
+import routePaths from '../../constants/routePaths';
+
+const SIGN_UP_USER = gql`
```

Comment:
you could create a constants file with all of the mutations and queries you will be using on the FE.